CONTRIBUTING.md

```diff
@@ -69,6 +69,10 @@ copying resources into the body of the commit over providing external links.
 Structure large commit messages with headers, references etc. Remember, however,
 that the commit message is always going to be rendered in plain text.
 
+Please add `[skip ci]` to the commit message when the change doesn't require a 
+compilation, such as documentation only changes.  This is required to be good
```

Comment:
Maybe ...such as documentation only changes, to avoid unnecessarily wasting the project's build resources.

?